### DISCUSSION TITLE
Welcome to Learning Path Discussions!

### DISCUSSION THREAD

**Discussion by jasonrandrews** [ArmDeveloperEcosystem/arm-learning-paths#64] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Arm Developer Program and discuss Learning Paths.

  Use this area if you 

  * are looking for feedback on your idea for a new Learning Path before you start. We can help you plan your new Learning Path.

  * have an idea for a new Learning Path, but are not able to create it. Please share it, and another community member may be able to create it. 
  
  * need help with the Learning Path creation process or have questions.

**Comment by devoneonone** [comment]
Thanks so much...

**Reply by devoneonone** [reply]
Yes, I am going here want I have bought, that was one month happened and I search but I don't find what I would like have. This Card is a 5 or 7 years that it was developed,  it's a PINE A64-LTS V2,  WITH 2 MEGA MEMORY ,4 ARM A53, MANY GPIO'S AND INTERNALS FUNCTIONS TO DISCOVER BUT I DON'T HAVE FIND A GOOD OPERATING SYSTEM FOR ARM64  THAT GIVE A EASY ACCES IN LOW LEVEL PROGRAMMINGTESTS ALL THIS ONES...
